AndroidTest/app/build.gradle

```diff
@@ -89,6 +89,8 @@ dependencies {
         force true
     }
 
+    compile 'com.google.code.findbugs:jsr305:3.0.0' //this is needed for some versions of android
```

Comment:
Nope, the test app uses standard compilation and brings in all the tests classes that way